tests/analog/message.test.ts

```diff
@@ -29,7 +29,210 @@ const downlinkMessages: TMessageExamples = {
             actual: 0xfe
         }
     },
-    'invalid correctTime2000': {
+    'setParameters + getParameters': {
+        bytes: getBytesFromHex('03 02 04 0c 03 0b 1a 52 b8 09 42 52 b8 2d 42 17 00 04 01 04 04 01 1a 63'),
```

Comment:
it would be good to separate different commands and LRC with two spaces

Reply:
done